content/posts/ytnapariparty.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "A yt-napari party"
+date: 2021-01-12T18:22:25-06:00
+author: Chris Havlin
```

Comment:
Do you have a personal webpage? We can link it here.

Reply:
yes... but it's empty... if I get to updating it soon I'll edit this PR but just publish without whenever. I can always submit a PR to update it?

Reply:
Yep! Totally ok to update later! I just wanted to make sure we *did* link it if you had one.